src/material/sort/sort-header.ts

```diff
@@ -130,7 +130,7 @@ export class MatSortHeader extends _MatSortHeaderBase
   @Input() arrowPosition: 'before' | 'after' = 'after';
 
   /** Overrides the sort start value of the containing MatSort for this MatSortable. */
-  @Input() start: 'asc' | 'desc';
+  @Input() start: SortDirectionExplicit;
```

Comment:
The current code in master seems incorrect to me. I would've expected it to be a `SortDirection`. @andrewseguin do you have more context on this?